pkg/ebpf/tracee.go

```diff
@@ -1252,11 +1290,17 @@ func (t *Tracee) writeProfilerStats(wr io.Writer) error {
 }
 
 func (t *Tracee) getProcessCtx(hostTid uint32) (procinfo.ProcessCtx, error) {
+
+	//FIXME: is this access going to affect performance?
+	mainBpfModule, ok := t.bpfModules[BPFModuleMain]
```

Comment:
nit: in some places you call this bpfMainModule and in some others mainBpfModule - let's try to stick with only one of them